### PR TITLE
remove a few steps needed for debugging

### PR DESCRIPTION
**Pull request by xster**
Make local debugging a bit easier. We're using local engine and never shipping it. Just make it expect arm64. Also change the flutter build to release to remove a footgun.

### CODE REVIEW

File: DoubleEngine.xcodeproj/xcshareddata/xcschemes/DoubleEngine.xcscheme [hunk 1]
```diff
@@ -31,7 +31,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```
Comment (gaaclarke) [inline DoubleEngine.xcodeproj/xcshareddata/xcschemes/DoubleEngine.xcscheme:34]: Can we revert these changes, please?

**Review by gaaclarke — APPROVED** [review]

File: DoubleEngine.xcodeproj/xcshareddata/xcschemes/DoubleEngine.xcscheme (hunk 1 above)
Comment (xster) [reply DoubleEngine.xcodeproj/xcshareddata/xcschemes/DoubleEngine.xcscheme:34]: it's intended, since the purpose is to check performance. 

Run is used to check the simple memory counter in Xcode, Profile is for memory in instruments.

Comment (gaaclarke) [reply DoubleEngine.xcodeproj/xcshareddata/xcschemes/DoubleEngine.xcscheme:34]: Ahh, I just use Profile, did you notice a difference between Release and Profile?

Comment (xster) [reply DoubleEngine.xcodeproj/xcshareddata/xcschemes/DoubleEngine.xcscheme:34]: There's the vm-service isolate in profile. What do you do in the Podfile?
